src/build_infrastructure/android/app.py

```diff
@@ -41,5 +46,18 @@
                                     description="CI Pipeline assets for amplify-android",
                                     env=TARGET_ENV)
 
+# Maven publisher stack
+maven_publisher_stack_props = {
+    'github_source': { 
+        'owner': github_owner,
+        'repo': REPO, 
+        'base_branch': branch,
+        'release_pr_branch': release_pr_branch
+    },
+    'codebuild_project_name_prefix': 'AmplifyAndroid'
```

Comment:
Any way to make this agnostic of project, so that we can use this script for all 3 of our repos?

Reply:
We'll need separate stacks, but we're reusing the CDK constructs.

When we setup the other projects, we'll add a `sdk_maven_publisher_stack_props` that will have a different CodeBuild project name prefix...something like `AndroidSDK`. Then we create the stack:

```python
MavenReleaseStack(app, "AndroidSDKMavenPublisher", sdk_maven_publisher_stack_props, description="Assets used for publishing aws-sdk-android to maven.", env=TARGET_ENV)
```